package.json

```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "@adobe/asset-compute-commons": "^1.1.2",
     "@adobe/cgroup-metrics": "^3.0.3",
-    "@adobe/httptransfer": "^2.1.1",
+    "@adobe/httptransfer": "^2.2.0",
```

Comment:
It will bring in his changes. But so would any reinstall of the SDK (in any place it's used), since the caret `^` means to grab any bugfix or feature update. I'm just making it explicit now. If we have to keep the version without changes, we would need to pin the version, or use the tilde `~` instead (tilde only grabs patches/bugfixes).

Reply:
Also, all unit and worker tests pass, so it looks pretty safe to get it in.

Reply:
@bobvanmanen Wdyt?